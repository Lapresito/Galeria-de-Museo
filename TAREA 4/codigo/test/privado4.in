crearGaleria 02/05/2023
obtenerVisitaDiaGaleria 01/05/2023
crearGrupoABB
crearVisitante 4 Andrea Blanco 25
agregarAGrupoABB
crearVisitante 10 Georgina Perez 20
agregarAGrupoABB
llegaGrupoGaleria 
crearPieza 1 LaMenina Pumino Rembrant
agregarPiezaGaleria
crearPieza 2 ElDesayuno Lobon VanGohns
agregarPiezaGaleria
crearExposicion 01/05/2023 20/05/2023 100 10
agregarExposicionGaleria
crearExposicion 08/05/2023 30/05/2023 100 11
agregarExposicionGaleria
agregarPiezaAExposicionGaleria 1 10
piezasEnExposicionGaleria
agregarPiezaAExposicionGaleria 2 10
piezasEnExposicionGaleria
avanzarAFechaGaleria 10/05/2023
piezasEnExposicionGaleria
crearPieza 3 LaPersistencia Osito Warhill
agregarPiezaGaleria
piezasEnReservaGaleria
crearVisitante 1 Gardenia Alvarez 22
indiceFelicidadVisitanteGaleria
agregarPiezasFavoritasVisitante 4 1 2 3 4
indiceFelicidadVisitanteGaleria
liberarVisitante
liberarGaleria
Fin