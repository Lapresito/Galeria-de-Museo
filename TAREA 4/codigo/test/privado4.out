Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Galeria creada con exito.
2> No existen registros de visitaDia para la fecha 1/5/2023
3> El grupoABB ha sido creado de forma exitosa.
4> Visitante creado de forma exitosa.
5> Se ha agregado el visitante al grupo de forma exitosa.
6> Visitante creado de forma exitosa.
7> Se ha agregado el visitante al grupo de forma exitosa.
8> Se ejecuto llegaGrupoTGaleria con exito.
9> Pieza creada en forma exitosa.
10> La pieza fue agregada a la galeria con exito.
11> Pieza creada en forma exitosa.
12> La pieza fue agregada a la galeria con exito.
13> Exposicion creada con exito.
14> La exposicion fue agregada a la galeria con exito.
15> Exposicion creada con exito.
16> La exposicion fue agregada a la galeria con exito.
17> La pieza de id 1 fue agregada a la exposicion de id 10.
18> Las piezas en exposicion en la galeria son:
1 
19> La pieza de id 2 fue agregada a la exposicion de id 10.
20> Las piezas en exposicion en la galeria son:
1 2 
21> Se avanzo la fecha de la galeria a 10/5/2023
22> Las piezas en exposicion en la galeria son:
1 2 
23> Pieza creada en forma exitosa.
24> La pieza fue agregada a la galeria con exito.
25> Las piezas en reserva en la galeria son:
3 
26> Visitante creado de forma exitosa.
27> El indice de felicidad del visitante es 1.00
28> Se ejecuto agregarPiezasFavoritasTVisitante exitosamente.
29> El indice de felicidad del visitante es 0.50
30> Visitante liberado con exito.
31> Galeria liberada con exito.
32> Fin.
